.github/workflows/test.yml

```diff
@@ -64,6 +64,9 @@ jobs:
       - name: Install Dependencies
         run: yarn --ignore-scripts
 
+      - name: Create .graphclient folder

```

Comment:
we should make sure this PR pulls from develop before merging because this might override the new workflow files.